javaparser-core/src/main/java/com/github/javaparser/ast/body/ModifierSet.java

```diff
@@ -65,34 +67,76 @@ public static AccessSpecifier getAccessSpecifier(int modifiers) {
         }
     }
 
+    public static AccessSpecifier getAccessSpecifier(NodeWithModifiers node) {
+        if (isPublic(node)){
+            return AccessSpecifier.PUBLIC;
+        } else if (isProtected(node)){
+            return AccessSpecifier.PROTECTED;
+        } else if (isPrivate(node)){
+            return AccessSpecifier.PRIVATE;
+        } else {
+            return AccessSpecifier.DEFAULT;
+        }
+    }
+
     public static int addModifier(int modifiers, int mod) {
         return modifiers | mod;
     }
 
+    @Deprecated
     public static boolean hasModifier(int modifiers, int modifier) {
         return (modifiers & modifier) != 0;
     }
 
+    public static boolean hasModifier(NodeWithModifiers node, int modifier) {
+        return (node.getModifiers() & modifier) != 0;
+    }
+
+    @Deprecated
```

Comment:
Can you please add a javadoc like

    /** @deprecated use {@link #hasModifier(...)} instead */

everywhere?